lib/methods/createMany.js

```diff
@@ -0,0 +1,30 @@
+var async = require('async');
+
+/**
+ * Creates multiple models.
+ *
+ * Utilizes batch operation if available or falls back to the default connector
+ * implementation and manually creates models one by one.
+ *
+ * @param {Arrow.Model} Model The model class being updated.
+ * @param {Array<Object>} values Array of attributes to set on the new models.
+ * @param {Function} callback Callback passed an Error object (or null if successful), and info about the batch operation.
+ * @throws {Error}
+ */
+exports.createMany = function (Model, values, callback) {
+	// Use batch delete if possible
+	var currentUser = this.user || this.baseUser;
+	var isAdminUser = currentUser.admin === 'true';
+	var modelSupportsBatchCreate = typeof Model.createMany === 'function';
```

Comment:
By some reason typeof Model.createMany is always undefined on my side, even for the "custom_object" model and also during the 'should create models via batch operation' unit test. Some idea why it is like that?